docs/_documentations/openshiftregistry.md

```diff
@@ -66,25 +66,23 @@ Some of the following instructions were adapted from [Remotely Push and Pull Con
       ```
 6. Copy the value from the **token** field.
 
-### Adding the OpenShift registry in Che
+### OpenShift registry in Che and Codewind
```

Comment:
I think we can remove `Che` from the title since we are no longer doing it on Che